src/filters/simple-filter-directive.js

```diff
@@ -183,13 +183,23 @@ angular.module('patternfly.filters').directive('pfSimpleFilter', function () {
         return foundFilter !== undefined;
       };
 
+      $scope.enforceSingleSelect = function (filter) {
+        if (filter.type === 'select') {
+          _.remove($scope.config.appliedFilters, {title: filter.title, type: filter.type});
+        }
+      };
+
       $scope.addFilter = function (field, value) {
         var newFilter = {
           id: field.id,
           title: field.title,
+          type: field.filterType,
           value: value
         };
         if (!$scope.filterExists(newFilter)) {
+
+          $scope.enforceSingleSelect(newFilter);
+
           $scope.config.appliedFilters.push(newFilter);
 
```

Comment:
I was wondering if we should be consistent and always show the order that the filters were added.  Removing old, and adding new, makes the filter appear last in the active filters list, so user can see what they added/changed last.
But, it might be better, as Jeff suggests, to just change the existing filter and it keeps it's position in the active filters list.